### ISSUE TITLE
Broken symmetric power L-function friends

### ISSUE THREAD

**Issue by roed314** [LMFDB/lmfdb#6245]
On elliptic curve pages, the "Symmetric square L-function" and "Symmetric cube L-function" links in the friends box are broken (see [14.a1](https://beta.lmfdb.org/EllipticCurve/Q/14/a/1) and its [isogeny class](https://beta.lmfdb.org/EllipticCurve/Q/14/a/) for example).  I think these have been disabled on purpose; if so, we should also remove the friend links.

**Comment by alozanoroble** [comment]
I also see the broken links. Were they working at some point? Also, do you refer to the "Related objects" box as the 'friends'? :)

**Comment by AndrewVSutherland** [comment]
Yes, related objects are friends.  And the properties box is the "Lady Gaga" box (I think credit/blame for this goes to @jvoight).
